.github/workflows/quality-checks.yml

```
@@ -16,7 +16,7 @@ jobs:
         os: [ubuntu-20.04]
         python-version:
           - "3.11"
-        node-version: [18]
+        node-version: [18, 20]
```

```suggestion
        node-version: [20]
```
Optional: You could save us a shard by just testing quality on Node 20. My experience with Python is that quality checking on the newest version is totally sufficient; if some code is "low quality" in the older Python version, newer Python isn't going to like it either. Not sure if that transfers to the Node world but I imagine it does.